agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataFetcher.java

```diff
@@ -116,7 +119,9 @@ public void prepareQuickPulseDataForSend(String redirectedEndpoint) {
       throw td;
     } catch (Throwable e) {
       try {
-        logger.error("Quick Pulse failed to prepare data for send", e);
+        try (MDC.MDCCloseable ignored = QUICK_PULSE_SEND_ERROR.makeActive()) {
+          logger.error("Quick Pulse failed to prepare data for send", e);
+        }
```

Comment:
have we seen this one in kusto?

Reply:
found 162 records

Reply:
can you send me one (with the exception) on teams?